src/main.cxx

```diff
@@ -1,14 +1,12 @@
 #include "ruby_data.hxx"
 #include "qt/main_window.hxx"
 
-#include <QtGui/QApplication>
+#include <QtWidgets/QApplication>
 #include <QtCore/QTextCodec>
 
 int main(int argc, char* argv[]) {
   QApplication app(argc, argv);
 
-  QTextCodec::setCodecForCStrings(QTextCodec::codecForName("UTF-8"));
```

Comment:
Is UTF-8 the default now in Qt5?

Reply:
`setCodecForCStrings` was removed in Qt5, and a solution someone suggested is to [save all source files in UTF-8](https://stackoverflow.com/questions/13968734/qtextcodecsetcodecforcstrings-is-gone-in-qt5) (so I'm assuming it's default). Btw I'm editing a game commented mostly in Japanese and it works as expected.